third_party/java/java_tools/update_java_tools.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# A script to update the checked-in jars corresponding to Java tools used
+# by the Java rules in Bazel.
+#
+# For usage please run
+# ~/third_party/java/java_tools/update_java_tools.sh help
+
+declare -a all_tools=("JavaBuilder" "VanillaJavaBuilder" "GenClass" "Runner" \
```

Comment:
Do you need this? Quoting the man page of bash:

_It is possible to obtain the keys (indices) of an array as well as the values.  `${!name[@]}` and `${!name[*]}`expand to the indices assigned in array variable name.  The treatment when in double quotes is similar to the expansion of the special parameters `@` and `*` within double quotes._

Reply:
Okay then. I added the variable because seeing all the `$!@` intimidated me, but if it's on the man page...